templates/style_guide.html

```diff
@@ -0,0 +1,428 @@
+{% extends "base.html" %}
+{% load i18n %}
+{% load static %}
+
+{% block title %}Style Guide{% endblock %}
+
+{% block content %}
+  <div class="py-0 px-3 mb-3 md:py-6 md:px-0" x-data="{
+		mode: localStorage.getItem('colorMode'),
```

Comment:
What's the purpose of this @4down ?

Reply:
![image](https://github.com/cppalliance/temp-site/assets/49487079/0623074b-f8d0-4c79-a950-223c3158bdfb)

It's there to give quick access to toggling back and forth between modes to see each element in light/dark modes.  I didn't want to  change anything outside of the style guide as much as possible so moving where the x-data is in the header wasn't something I wanted to attempt.

Reply:
(it's for that sun icon which changes to the moon there in the images..  @gregnewman let me know if you think there's an issue with the way I implemented it though!)